lib/ansible/modules/packaging/os/satellite_subscription.py

```diff
@@ -0,0 +1,441 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# (c) 2019, Luc Stroobant (luc.stroobant@wdc.com)
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+ANSIBLE_METADATA = {"metadata_version": "1.1", "status": ["preview"], "supported_by": "community"}
+
+
+DOCUMENTATION = """
+---
+module: satellite_subscription
+short_description: Red Hat Satellite subcription (license) mangement module
+description:
+  - This module allows to assign or remove subscriptions to hosts in the satellite
```

Comment:
Always use trailing dots, names should be capitalized.
I feel like a teacher in elementary school now ;-)
```suggestion
  - This module allows to assign or remove subscriptions to hosts in Red Hat Satellite.
```

Reply:
Sorry. :-)